### PR TITLE
Upgrade selenium version 

### PR DESCRIPTION
**Pull request by sherrif10**
upgrade selenium version to latest version.

### CODE REVIEW

**Comment by kdaud** [comment]
@sherrif10 does this solve the current problem?

**Comment by jwnasambu** [comment]
@sherrif10 the code LGTM.  Though we miss the description of what you changed.

**Comment by kdaud** [comment]
Which problem is being addressed here?

**Comment by sherrif10** [comment]
We should endeavor using latest version of selenium.thate the idea behind it.

File: qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/helper/Page.java [hunk 1]
```diff
@@ -359,7 +359,7 @@ public void waitForElement(By by) {
 		waiter.until(ExpectedConditions.visibilityOfElementLocated(by));
 	}
 
-	public void waitForElementWithSpecifiedMaxTimeout(By by, long secs) {
+	public void waitForElementWithSpecifiedMaxTimeout(By by, Duration secs) {
```
Comment (kdaud) [inline qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/helper/Page.java:362]: Is this change part of the selenium  version upgrade?

**Pull request merged**